specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/componentWorkItemConfigs_API.json

```diff
@@ -238,11 +314,14 @@
           "description": "Serialized JSON object for detailed properties"
         },
         "ValidateOnly": {
-          "type": "boolean",
-          "description": "Boolean indicating validate only"
+          "type": "string",
+          "description": "True or false string indicating validate only"
```

Comment:
Why are you changing a boolean property to a string one that takes "true" or "false"?

This would also be a breaking change to an API version marked as stable.

Reply:
I'm conflicted here since I wanted to make this represent what is being sent over the wire.

When capturing an example how the way Ibiza submits the data I noticed that we have been sending a string. When looking at our code we parse a string as a Boolean and return a Boolean as a string; however the model is using a Boolean. 

**Anyway, I'll revert this change since it's been this way for a very long time.**